gitconfig

```diff
@@ -30,8 +30,11 @@
 	default = simple
 [core]
 	autocrlf = input
-        excludesfile = ~/.gitignore 
+	excludesfile = ~/.gitignore
 	precomposeunicode = true
+    	# attributesFile = ~/.gitattributes # better to check attributes file into repo that use a global one
```

Comment:
Looks like this is commented out? Let me know if you can commit this change!

Reply:
Yeah. The new hotness is to include a .gitattributes file _in each project repo_ to both tailor it and so that all contributors have the same one. I put this in and commented out as a reminder that these settings _shouldn't_ be in a devs homedir